tests/cluster.test.sh

```
@@ -223,6 +228,29 @@ else
   echo "  FAIL  C10.rfc-present"; FAIL=$((FAIL+1))
 fi
 
+echo "== C11: RFC 0012 §3.11 light-R5 shape locks =="
+# (a) Phase 0 persists RUN_DIR_ABS as pointer line 2 (the bare-RUN_ID pointer
+# left fresh-shell phases probing a $UBERDEV_TMPDIR path that can never exist).
+assert_grep "$SKILL" "printf '%s\\\\n%s\\\\n' \"\\\$RUN_ID\" \"\\\$RUN_DIR_ABS\"" \
+  "C11.pointer-carries-run-dir-abs"
+assert_no_grep "$SKILL" 'RUN_DIR="\$UBERDEV_TMPDIR/\.uberdev/cluster/\$RUN_ID"' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)